### PR TITLE
PYIC-3342: CIMIT Stubs: Update `putContraIndicators` stub to add CIs from supplied VC

### PR DESCRIPTION
**Pull request by TalatAydinGDS**
### What changed

CIMIT Stubs: Update `putContraIndicators` stub to add CIs from supplied VC

### Why did it change

Update putContraIndicators stub to extract any CIs from the supplied VC and add/update the CIs stored for that user on the CIMIT stub ddb table. The logic for this should be a simplified work-alike of that on the actual CIMIT putContraIndicators, viz:

### Issue tracking

- [PYIC-3342](https://govukverify.atlassian.net/browse/PYIC-3342)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-3342]: https://govukverify.atlassian.net/browse/PYIC-3342?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/PutContraIndicatorsHandler.java [hunk 1]
```diff
@@ -1,23 +1,60 @@
...
-        return gson.toJson(PutContraIndicatorsResponse.builder().result("success").build());
+    public void handleRequest(InputStream input, OutputStream output, Context context)
+            throws IOException {
+        LOGGER.info(new StringMapMessage().with("Function invoked:", "PutContraIndicators"));
+        PutContraIndicatorsRequest putContraIndicatorsRequest;
+        PutContraIndicatorsResponse response;
+
+        try {
+            putContraIndicatorsRequest = mapper.readValue(input, PutContraIndicatorsRequest.class);
+        } catch (Exception ex) {
```
Comment (Wynndow) [inline di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/PutContraIndicatorsHandler.java]:
```suggestion
        } catch (IOException ex) {
```
Might be worth being more specific with this catch

File: di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/service/ContraIndicatorsService.java [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        }
+    }
+
+    private ContraIndicatorEvidenceDto parseContraIndicatorEvidence(SignedJWT signedJWT)
+            throws CiPutException {
+        JSONObject vcClaim;
+        try {
+            vcClaim = (JSONObject) signedJWT.getJWTClaimsSet().getClaim(VC_CLAIM);
+        } catch (ParseException e) {
+            String message = "Failed to parse ContraIndicators response json";
```
Comment (Wynndow) [inline di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/service/ContraIndicatorsService.java]:
```suggestion
            String message = "Failed to parse VC claim";
```

**Review by Wynndow — COMMENTED** [review]
Looks good, just a couple of really small things.

File: di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/service/ContraIndicatorsService.java (hunk 2 above)
Comment (TalatAydinGDS) [reply di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/service/ContraIndicatorsService.java]: Done.

File: di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/PutContraIndicatorsHandler.java (hunk 1 above)
Comment (TalatAydinGDS) [reply di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/PutContraIndicatorsHandler.java]: Done.

**Review by Wynndow — APPROVED** [review]

**Pull request merged**